flink-ml-benchmark/src/main/resources/minmaxscaler-benchmark.json

```diff
@@ -22,19 +22,15 @@
         "vectorDim": 100,
         "colNames": [
           [
-            "features"
+            "inputCol"
```

Comment:
Can you verify that this configuration still works after the change?

Same for other configuration files updated by this PR.

Reply:
Thanks for reminding, I verified all updated files and they work well.